src/utils/number.ts

```diff
@@ -0,0 +1,14 @@
+const START_EXPONENT = 1e-7;
+
+export const getDigits = (x: number) => {
+  if (x <= START_EXPONENT) {
```

Comment:
`Number.EPSILON` fits better here, if i got it right

Reply:
Number.EPSILON is too small for this case. The code should catch exactly exponent.